fakebook/src/app/models/user.ts

```diff
@@ -0,0 +1,10 @@
+export interface User {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string | undefined;
+  profilePictureUrl: string | null;
```

Comment:
Slight nitpick, but is the union type `string | undefined` more appropriate than `string | null` for `profilePictureUrl`?